src/locales/en.json

```
@@ -1,4 +1,10 @@
 {
+  "404": {
+    "title": "The content you’re looking for seems to have disappeared.",
+    "main": "Return to {link} or search for content with the field below.",
```

The text changed in the mockup:
```suggestion
    "main": "Go to {link} or search for something similar from the field below.",
```

---

@krysal @panchovm I'm not sure about this text. The `{link}` goes to the home page, so 'return to' makes sense to me, and 'search for something similar' doesn't make a lot of sense to me. Similar to what?

---

If someone comes from another website to this page the "Go to ..." seems more appropriate, the "Return" makes me think of the go back feature of the browser, but maybe that is just me.

About _something similar_, I was assuming an image or audio that it's no longer shown for been taken down or whatever reason, but none of these are big deals neither.

---

Both of those explanations make sense, thank you!